### PR TITLE
Reverts crybaby inducer also does the sync things

### PR DESCRIPTION
**Pull request by Poojawa**
does a better job than #5829 at reverting hugbox removal. Can't let anyone actually *work* for something. why play a game then?

### CODE REVIEW

**Comment by deathride58** [comment]
You created the branch from the wrong branch you dork

**Comment by Poojawa** [comment]
whatever.

**Comment by JogBoba** [comment]
If you give the console part it’s own pr and leave the piping I’d be all for it. Thank you for creating a proper revert.

**Comment by Poojawa** [comment]
This also fixes the whole ghost roles can't talk thing too.

File: .gitignore [hunk 1]
```diff
@@ -198,4 +198,3 @@ tools/MapAtmosFixer/MapAtmosFixer/obj/x86/Debug/MapAtmosFixer.csproj.CoreCompile
 
 #GitHub Atom
 .atom-build.json
-cfg/admin.txt
```
Comment (deathride58) [inline .gitignore:201]: Send to upstream

File: cfg/admin.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 poojawa role=admin
+junozumori role=admin
```
Comment (deathride58) [inline cfg/admin.txt]: who

**Review by deathride58 — DISMISSED** [review]
Remind me to remake the xenobio console change as a separate PR. That change is being received surprisingly well on-server.

**Comment by Poojawa** [comment]
>That change is being received surprisingly well on-server.

dugba4qw0aa9aoc

**Review by deathride58 — APPROVED** [review]
You really should have done the sync separately from the xenobio console revert

**Pull request merged**